.infrastructure/00_main.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  required_version = "1.2.8"
+
+  backend "s3" {}
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.67.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.aws_region
+  default_tags {
+    tags = var.tags
+  }
+}
+
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+
+  default_tags {
+    tags = var.tags
+  }
+}
```

Comment:
I think we don't need this provider.
Do we have some resources located in "us-east-1" zone that are needed in this PR?

Reply:
corretto, lascio solo la region definita nelle variables con default=eu-south-1